src/index.js

```diff
@@ -53,6 +54,8 @@ class Node extends EventEmitter {
       this.swarm.on('peer-mux-closed', (peerInfo) => {
         this.emit('peer:disconnect', peerInfo)
       })
+
+      this.modules.transport.push(new Circuit(this.swarm, _options.relay))
```

Comment:
This is probably not the best place to instantiate `Circuit`, I would much rather move it to IPFS where the `libp2p` node is set up and instantiate it there, but since it depends on `libp2p-swarm` it would require moving swarm instantiation there too.